react/features/base/config/constants.js

```diff
@@ -44,5 +44,6 @@ export const TOOLBAR_BUTTONS = [
     'stats',
     'tileview',
     'toggle-camera',
-    'videoquality'
+    'videoquality',
+    'poll'
```

Comment:
Can we have this in alphabetical order? https://github.com/jitsi/jitsi-meet/blob/master/CONTRIBUTING.md#formatting